WeakAuras/GenericTrigger.lua

```diff
@@ -86,6 +86,7 @@ local debug = WeakAuras.debug;
 local events = WeakAuras.events;
 local loaded_events = WeakAuras.loaded_events;
 local loaded_unit_events = {};
+local loaded_periodic_events = {};
```

Comment:
loaded_periodic_events is a bad name, I think. These aren't events, but callbacks on intervals. Feel free to bikeshed this some.

Reply:
Yeah, I was thinking about this too. In the end my thought process was, you (currently) write it in the events box, so why not just call them events. They're very similar to events, the main difference being that they're fired at different times for different triggers.

I wouldn't be opposed to changing the name though, maybe `loaded_periodic_callbacks` or `loaded_periodic_tickers` or something.

Reply:
Changed to `loaded_periodic_updates` (along with changing all other names containing `periodic_events` -> `periodic_updates`)